catroid/src/main/java/org/catrobat/catroid/stage/StageActivity.java

```diff
@@ -127,6 +128,8 @@ public void onCreate(Bundle savedInstanceState) {
 		stageDialog = new StageDialog(this, stageListener, R.style.stage_dialog);
 		calculateScreenSizes();
 
+		axesColor = getContext().getResources().getColor(R.color.axis_font_color);
```

Comment:
so is this really working? since new StageListener is called in 127, while the color is set here.

add the color to the constructor of the stage listener, initilialize the color there, and dont even save it here in the stage activity since its never again used (besides switching the scene, which is again handled in the stage listener anyways.)

and since youre already touching this, please remove below comment, and make a new final int NUMBER_OF_BITS_PER_COLOR_CHANNEL = 8; and use it instead of the hardcoded 8...